### PR TITLE
Add quickstarts for the catalog connectors

### PR DESCRIPTION
**Pull request by phillipleblanc**
## 🗣 Description

Adds quickstarts for the Catalog Connectors feature added in https://github.com/spiceai/spiceai/issues/1810

## 🔨 Related Issues

https://github.com/spiceai/spiceai/issues/1810

### CODE REVIEW

File: catalogs/databricks/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Databricks Unity Catalog Connector
+
+The Databricks Unity Catalog Connector allows you to connect to your tables in a Databricks Unity Catalog and query them using Spice.
```
Comment (lukekim) [inline catalogs/databricks/README.md]:
```suggestion
The Databricks Unity Catalog Connector makes querying Databricks Unity Catalog tables in Spice simple.
```

File: catalogs/databricks/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Step 6. Query a dataset
+
+```bash
+spice sql
+```
+
+```sql
+SELECT * FROM db_uc.<SCHEMA_NAME>.<TABLE_NAME> LIMIT 10;
+```
```
Comment (lukekim) [inline catalogs/databricks/README.md:91]: Add example output.

File: catalogs/spiceai/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Spice.ai Cloud Platform Catalog Connector
+
+The Spice.ai Cloud Platform Catalog Connector enables you to start querying the 200+ built-in datasets available in the Spice.ai Cloud Platform.
```
Comment (lukekim) [inline catalogs/spiceai/README.md]:
```suggestion
The Spice.ai Cloud Platform Catalog Connector makes querying datasets in the Spice.ai Cloud Platform simple including to the [200+ built-in community datasets](https://docs.spice.ai/building-blocks/datasets).
```

File: catalogs/spiceai/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+## Step 7. Explore the available datasets
+
+Use `show tables` in the Spice SQL REPL to see the available datasets.
+
+```sql
+show tables;
+```
+
```
Comment (lukekim) [inline catalogs/spiceai/README.md:86]: Example output.

File: catalogs/spiceai/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+Specify an `include` filter to limit the tables registered in the catalog.
+
+```yaml
+catalogs:
+  - name: spiceai
+    from: spiceai
+    include:
+      - tpch.*
+```
```
Comment (lukekim) [inline catalogs/spiceai/README.md:97]: Show tables again with example output.

File: catalogs/unity_catalog/README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+# Unity Catalog Connector
+
+The Unity Catalog Connector allows you to connect to your tables in a Unity Catalog and query them using Spice.
```
Comment (lukekim) [inline catalogs/unity_catalog/README.md]:
```suggestion
The Unity Catalog Connector makes querying tables in a Unity Catalog with Spice simple.
```

File: catalogs/unity_catalog/README.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Note: This is for the [open-source version of Unity Catalog](https://www.unitycatalog.io/). For connecting to the Databricks Unity Catalog Connector, see the [Databricks Unity Catalog Connector](../databricks/README.md).
```
Comment (lukekim) [inline catalogs/unity_catalog/README.md]:
```suggestion
Note: This quickstart applies to the [open-source version of Unity Catalog](https://www.unitycatalog.io/). To get started with the Databricks Unity Catalog Connector, see the [Databricks Unity Catalog Connector quickstart](../databricks/README.md).
```

File: catalogs/unity_catalog/README.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+## Step 6. Query a dataset
+
+```bash
+spice sql
+```
+
+```sql
+SELECT * FROM uc.<SCHEMA_NAME>.<TABLE_NAME> LIMIT 10;
```
Comment (lukekim) [inline catalogs/unity_catalog/README.md:60]: Example output.

**Review by lukekim — DISMISSED** [review]

File: catalogs/spiceai/README.md (hunk 4 above)
Comment (phillipleblanc) [reply catalogs/spiceai/README.md:86]: Added

File: catalogs/databricks/README.md (hunk 2 above)
Comment (phillipleblanc) [reply catalogs/databricks/README.md:91]: Added

File: catalogs/spiceai/README.md (hunk 5 above)
Comment (phillipleblanc) [reply catalogs/spiceai/README.md:97]: Added

File: catalogs/unity_catalog/README.md (hunk 8 above)
Comment (phillipleblanc) [reply catalogs/unity_catalog/README.md:60]: Added

File: catalogs/databricks/README.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
+# Databricks Unity Catalog Connector
+
+The Databricks Unity Catalog Connector makes querying Databricks Unity Catalog tables in Spice simple.
+
+## Prerequisites
+
+- A Databricks account with a Unity Catalog configured with 1+ tables. (see the [Databricks documentation](https://docs.databricks.com/en/data-governance/unity-catalog/index.html) for more information).
```
Comment (digadeesh) [inline catalogs/databricks/README.md]:
```suggestion
- A Databricks account with a Unity Catalog configured with 1 or more tables. (see the [Databricks documentation](https://docs.databricks.com/en/data-governance/unity-catalog/index.html) for more information).
```

File: catalogs/databricks/README.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+Add the following configuration to your `spicepod.yaml`:
+
+```yaml
+catalogs:
+  - name: databricks:<CATALOG_NAME>
+    from: db_uc
+    params:
+      endpoint: <instance-id>.cloud.databricks.com
+      mode: spark_connect # or delta_lake
+      databricks_cluster_id: 0711-151224-1rh2uz8m
```
Comment (ewgenius) [inline catalogs/databricks/README.md]:
```suggestion
      databricks_cluster_id: <cluster-id>
```

**Review by digadeesh — APPROVED** [review]

**Review by ewgenius — APPROVED** [review]

**Pull request merged**